### PR TITLE
Add documentation on the PDF tools

### PR DESCRIPTION
**Pull request by Technius**
Here is some documentation I wrote on:
* Some of the features used for interacting with PDFs
* Pitfalls/subtleties when using annotating PDFs
* The new PDF toolbox and the select PDF text tools

The instructions on "Exporting to PDF" are still WIP.

### CODE REVIEW

**Comment by Technius** [comment]
Merging in 24 hours if there are no objections.

File: docs/guide/pdfs.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+Due to the way that Xournal++ deals with PDF files, there are several
+limitations on the export:
+
+* Once you export annotations to PDF format, the annotations in the PDF file
+  cannot be edited. This is because the PDF format is not designed to be edited.
+* Xournal++'s PDF export does not preserve annotations that are built-in to the
+  PDF file, such as pop-up notes and forms.
+* Xournal++ will not allow you to override the background PDF when you export.
+  This is because overriding the background PDF will cause the annotations in
+  the journal will become inconsistent with the actual contents of the (newly
```
Comment (rolandlo) [inline docs/guide/pdfs.md:105]:
```suggestion
  the journal become inconsistent with the actual contents of the (newly
```

File: docs/guide/tools/pdf_tools.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Interacting with Selected Text
+
+Once text selection is finalized, a popup window with several actions will be
+shown.
+
+* The large button with the copy icon will copy the selected text to the clipboard.
+* The three buttons on the right side of the toolbox will create a stroke that
+  (from top to bottom, respectively): 1) highlights the selected text; 2)
+  underlines the selected text; 3) strikes through the selected text.
```
Comment (rolandlo) [inline docs/guide/tools/pdf_tools.md:43]: It would be nice to see the icons here.

File: docs/guide/pdfs.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Together, this means that you can savely "move" a `.xopp` file that has a PDF
+background _as long as you ensure that you "move" the file using `File > Save
+As`_!
+
+!!! bug
+    In Xournal++ 1.1.1 and older, there was a bug where attach mode settings
+        would not be remembered when loading. This bug has been fixed in Xournal++
+    1.1.2.
+
```
Comment (rolandlo) [inline docs/guide/pdfs.md:49]: Spaces should be removed from the line.
```suggestion

```

File: docs/guide/pdfs.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+To search for text contained in a PDF background page, use `Edit > Search` or
+the keyboard shortcut <kbd>Ctrl</kbd><kbd>F</kbd>. This will open a prompt that
+will allow you to search for the given text on all PDF pages.
+
+## Interacting with PDF Pages
+
+!!! note
+    This feature is only available in the nightly version.
+
```
Comment (rolandlo) [inline docs/guide/pdfs.md:81]: Here as well
```suggestion

```

File: docs/guide/tools/pdf_tools.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# PDF Tools
+
+!!! note
+    This feature is only available in the nightly version.
+
+The PDF tools can be used to interact with the background PDF.
+
+## PDF Toolbox
+
```
Comment (rolandlo) [inline docs/guide/tools/pdf_tools.md:9]: Here as well:
```suggestion

```

**Review by rolandlo — CHANGES_REQUESTED** [review]
Overall it looks good to me. Some minor issues are marked (including a wrong link).

**Review by rolandlo — APPROVED** [review]
LGTM

**Pull request merged**